packages/ui-keyring/src/account/changePassword.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017-2018 @polkadot/ui-keyring authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { TranslationFunction } from 'i18next';
+import { AccountResponse, State } from '../types';
+
+import updateAccount from './update';
+
+export default function changeAccountPassword (state: State, t: TranslationFunction, address: string, password: string, newPassword: string): AccountResponse {
+  const { keyring } = state;
+  let response = { error: undefined };
+
+  if (!address || !password || !newPassword) {
+    response.error = t('editor.change.password.error.missing.address.password', {
```

Comment:
What does keyring have to do with editor? This is keyring. Additionally, should translation be done, it is in the `ui-keyring` namespace, i.e. namespace is specific to the package. (No cross-sharing)